src/FixedDataTable.js

```diff
@@ -872,6 +872,7 @@ class FixedDataTable extends React.Component {
           fixedColumnsWidth={this.props.fixedColumnsWidth}
           fixedRightColumnsWidth={this.props.fixedRightColumnsWidth}
           isGroupHeader={true}
+          shouldUseLegacyComponents={this.props.shouldUseLegacyComponents}
```

Comment:
Declare proptype for `shouldUseLegacyComponents`  above.

Reply:
Done